mcs/class/corlib/Test/System.Threading.Tasks/TaskTest.cs

```diff
@@ -312,7 +312,7 @@ public void WaitAll_ManyTasks ()
 				}
 				AddToCleanup (tasks);
```

Comment:
unrelated, but this call seems unnecessary since we're waiting on the tasks below anyway, same as what AddToCleanup() ultimately does in the Teardown method, but with this we're holding on to 60*2000 task objects until the test suite is done.